README.md

```diff
@@ -126,7 +126,7 @@ final class CountingUpAlwaysEndInAHigherCount implements Property
         return $counter->current() < 100; // since upper bound is 100
     }
 
-    public function ensureHelBy(object $counter): object
+    public function ensureHeldBy(object $counter): object
```

Comment:
Not sure about this one :thinking:

Reply:
yep it's `ensureHeldBy ` in the interface 👍